.travis.yml

```diff
@@ -0,0 +1,18 @@
+script:
+- echo "hello world"
+- docker build --target development --tag todo-app:dev .
+- docker build --target production --tag todo-app:prod .
+- docker build --target test --tag todo-app:test .
+- docker run todo-app:test tests/test_viewmodel.py
+- docker run --env-file .env.test todo-app:test tests/test_app.py
```

Comment:
This setup is OK for now, but will become more painful as you add more testing files as you'll need to add a new command for each. What we might do is instead group any that require fake credentials, and run those with `docker run --env-file .env.test todo-app:test tests` and then group the ones with live credentials `docker run --env-file .env todo-app:test live_tests`  (or some better name!)

Reply:
I agree - never thought of that! Will add to my list - to be addressed as a stretch goal later in the program.